### ISSUE TITLE
Duration will not parse its own ISO format when weeks are added

### ISSUE THREAD

**Issue by markatom** [moment/luxon#482]
Hello 👋

I have an issue with recently introduced week tokens in duration ISO format. Consider the following code:

```js
Duration.fromISO('P8DT13H23M34S'); // valid
Duration.fromISO('P1W1DT13H23M34S'); // the input "P1W1DT13H23M34S" can't be parsed as ISO 8601
```

I understand that week tokens may not be supported in parsing function, however, the issue appears also when manipulating with the duration, as illustrated in the following code:

```js
Duration.fromISO(Duration.fromISO('P1DT13H23M34S').plus({weeks: 1}).toISO()); // the input "P1W1DT13H23M34S" can't be parsed as ISO 8601
```

This behavior could cause problems when serializing/deserializing durations.

Thank you for your awesome work on Luxon!

**Comment by icambron** [comment]
Yeah, we should support W in parsing too

**Comment by icambron** [comment]
Fixed in #489
